Mission involves creep.lua

```diff
@@ -0,0 +1,86 @@
+local argsUtil = require("Module:ArgsUtil")
+local cargoUtil = require("Module:CargoUtil")
+
+local p = {}
+
+-- kill ranges based on difficulty level
+local killRanges = {
+    easy = { 4, 12 },
+    hard = { 2, 5 },
+    special = { 4, 8 }
+}
+
+-- Function to calculate the reward range based on the kill range
+local function calculateRewardRange(lowerBound, upperBound)
+    local lowerReward = 20 * lowerBound * 1.5
+    local upperReward = 20 * upperBound * 1.5
+    return lowerReward .. "-" .. upperReward
+end
+
+function p.main(frame)
+    local args = argsUtil.merge(frame.args, frame:getParent().args)
+    local creepName = args[1] or mw.title.getCurrentTitle().text
+
+    -- Perform the missions Cargo query
+    local missionQueryOptions = {
+        tables = "Missions, missionNPCs",
+        fields = "Missions._pageTitle, Missions.name, Missions.missionID, missionNPCs.npc",
+        join = "Missions.missionID=missionNPCs.missionID",
+        where = "Missions.missionID LIKE 'T_KillEnemies%'",
+        groupBy = "",
```

Comment:
no need to put empty string for the ones you arent using, you can just leave them out (up to you though)